config/development.yml.erb

```diff
@@ -64,6 +64,7 @@ jwk_private_key_data:
 
 openai_chat_completion_api_key: ''
 openai_evaluate_rubric_api_key:
+openai_aichat_safety_api_key: ''
 
```

Comment:
Why is this not marked as `!Secret`? That way anyone can try this in dev without having to get a special dev key for their local usage.

Also, the same should added to adhoc, see: https://github.com/code-dot-org/code-dot-org/pull/61259/files